tests/backup/backup_restore_basic_test.go

```diff
@@ -3842,3 +3843,333 @@ var _ = Describe("{KubeAndPxNamespacesSkipOnAllNSBackup}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// ExcludeDirectoryFileBackup excludes mentioned directories or files from backed-up apps and restores them.
```

Comment:
Can you create another file called `backup_kdmp_test.go` and add this test there?